.github/workflows/merge.yml

```diff
@@ -183,6 +183,15 @@ jobs:
         env:
           cmd_type: UNIT_TEST
 
+      - name: Blackduck Scan
+        uses: lumada-common-services/gh-composite-actions@1.2.0
+        env:
+          BlackDuck_Project_Version: ${{ steps.version.outputs.current-version }}
+          BlackDuck_Project_Name: "${{ inputs.sonar_project_key }}"
+          BlackDuck_Api_Token: "${{ secrets.BLACKDUCK_TOKEN }}"
+          BlackDuck_Url: "${{ vars.BLACKDUCK_URL }}"
+          ADDITIONAL_ARGS: --detect.accuracy.required=NONE # to fix gradle build error https://community.synopsys.com/s/article/Scan-fails-with-FAILURE-ACCURACY-NOT-MET-error
```

Comment:
`actions-common` is aimed to be a central place for Pentaho builds - that are in Maven. Does this `ADDITIONAL_ARGS` makes sense still?

Reply:
I have removed additional arguments for now, it can be added back if needed.

Reply:
We can have an `input` to pass different arguments if needed. Something like:
```
...
ADDITIONAL_ARGS: ${{ inputs.blackduck_additional_args }}
```